packages/node/src/evm/fulfillments/index.ts

```diff
@@ -1,4 +1,5 @@
-import flatMap from 'lodash/flatMap';
+import { Transaction } from 'ethers';
+import { flatten } from 'lodash';
```

Comment:
Maybe we should look into setting up some sort of linting rule to check for: `from 'lodash';` 😄

Reply:
I like that idea :)

btw. does it impact airnode at all? I mean the startup will be a bit slower, but I don't think this has as big impact as when compared to FE.

Reply:
I'm not actually sure. I imagine it will just make the deployed code a bit larger but that would have a negligible impact on the performance.

Definitely not as big an impact as on FE code though